siaskynet/_upload.py

```diff
@@ -48,41 +109,12 @@ def upload_file_request(self, path, custom_opts=None):
         return None
 
     with open(path, 'rb') as file_h:
-        filename = opts['custom_filename'] if opts['custom_filename'] \
-            else os.path.basename(file_h.name)
-        files = {opts['portal_file_fieldname']: (filename, file_h)}
+        if not opts['custom_filename']:
+            opts['custom_filename'] = os.path.basename(file_h.name)
 
-        return self.execute_request(
-            "POST",
-            opts,
-            files=files,
-        )
+        upload_data = {opts['custom_filename']: file_h}
 
-
-def upload_file_request_with_chunks(self, path, custom_opts=None):
```

Comment:
We can't delete any methods because it would break backwards compatibility. I would keep this and just have it call the new method

Reply:
i can do this but i think there are compatibility issues elsewhere too; i originally made this PR while porting my code to the newer interfaces.